sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StateStoreSuite.scala

```diff
@@ -1025,6 +1003,51 @@ abstract class StateStoreSuiteBase[ProviderClass <: StateStoreProvider]
       }
     }
   }
+
+  def getDefaultSQLConf(minDeltasForSnapshot: Int, numOfVersToRetainInMemory: Int): SQLConf
+  def getDefaultStoreConf(): StateStoreConf = StateStoreConf.empty
+
+  protected def fileExists(
+      provider: ProviderClass,
```

Comment:
Nice! +1 for this change from `HDFSBackedStateStoreProvider` to `ProviderClass`.